doc/webui/README.md

```diff
@@ -24,46 +26,233 @@ single instances can be configured independently.
 The configuration view of elektra web is similar to the tree view of the
 [qt-gui](https://github.com/ElektraInitiative/libelektra/tree/master/src/tools/qt-gui).
 
-![https://cdn.rawgit.com/ElektraInitiative/libelektra/http-api-proposal/doc/webui/ui_structure.png](https://cdn.rawgit.com/ElektraInitiative/libelektra/http-api-proposal/doc/webui/ui_structure.png)
+![https://cdn.rawgit.com/omnidan/libelektra/http-api-proposal/doc/webui/ui_structure.png](https://cdn.rawgit.com/omnidan/libelektra/http-api-proposal/doc/webui/ui_structure.png)
 
 
 ## API
 
-![https://cdn.rawgit.com/ElektraInitiative/libelektra/http-api-proposal/doc/webui/daemon_structure.png](https://cdn.rawgit.com/ElektraInitiative/libelektra/http-api-proposal/doc/webui/daemon_structure.png)
+![https://cdn.rawgit.com/omnidan/libelektra/http-api-proposal/doc/webui/daemon_structure.png](https://cdn.rawgit.com/omnidan/libelektra/http-api-proposal/doc/webui/daemon_structure.png)
+
+### Group elektrad API
 
 To access single instances, each elektra daemon (`elektrad`) provides a RESTful
 HTTP API:
 
- * **GET /version** - get `elektrad` version
- * **GET /kdb/:path** - get `path` configuration (same as `kdb get path`)
- * **POST /kdb/:path** - edit `path` configuration (same as `kdb set path`)
+#### GET /version
+
+get API and elektra version
+
++ Response 200 (application/json)
+
+        {
+            "api": 1,
+            "elektra": "0.8.17"
+        }
```

Comment:
Good idea! I just looked into attributes and data structures and switched all parts of the api to it, much more flexible and no redundancies :ok_hand: